### ISSUE TITLE
CSV category (Internal Sales Volume)

### ISSUE THREAD

**Issue by jordan31bit** [ArkadiusTradeTools/arkadius-trade-tools#50]
Internal Sales Volume is a category that is confusing to me. The number seems as if its representing gold amount but the term volume makes me think of quantity.  Could this be clarified?

Also, it seems the CSV file is already ordered from largest revenue to lowest. Does ATT do this automatically and does this incur a performance hit?

**Comment by timothymclane** [comment]
You're right that the nomenclature is misleading. It's a relic of ATT's internal verbiage and should probably be updated.

There is a performance hit for sorting, but it's so negligible as to be not worth considering.
